objects/LuaScript.ttslua

```diff
@@ -984,7 +989,8 @@ function setupGame()
     wait(0.1)
     setupStartingResources()
     wait(0.1)
-    setupInProgress = false
+    setupCampaignMisc()
```

Comment:
Is the `setupInProgress = false` line intentionally removed?

Reply:
Yes, the setupCampaignMisc() function will set that variable to false at the end